demos/python_demos/whiteboard_inpainting_demo/utils/ie_tools.py

```diff
@@ -0,0 +1,92 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import logging as log
+import numpy as np
+import os
+import sys
+
+from openvino.inference_engine import IENetwork, IECore # pylint: disable=import-error,E0611
+
+
+class IEModel:
+    """Class for inference of models in the Inference Engine format"""
+    def __init__(self, ie, model_path, conf=.6, device='CPU', ext_path='', num_reqs=1):
```

Comment:
`IEModel` can't utilize more than one  `num_reqs`. This argument should be removed. The same is for `load_ie_model()` and all subclasses.

Reply:
Fixed.